backend/app/api/game.py

```diff
@@ -157,10 +157,25 @@ def get_tile():
 
     return tile
 
-
+"""function to filter chat messages"""
 def filter_message(data: Dict[str, str]) -> Dict[str, str]:
-    """function to filter chat messages"""
```

Comment:
the Docstrings are meant to be just inside the function, not ontop. This has a specific meaning in Python